### DISCUSSION TITLE
QR code for 4K 9:16

### DISCUSSION THREAD

**Discussion by JerryDiver** [gopro/labs#1046] [Feature Requests]
![image](https://github.com/user-attachments/assets/1a3dba3c-dc4b-4286-b071-4a19ec4b3d93)

Gopro provides these codes for resolutions.
The Hero 12 supports 9:16 aspect 
Is there a code for it so I can use it in QR code?
fi 4K 9:16

**Comment by dnewman-gpsw** [comment]
I've added this to Issues, as there is no command.

**Comment by dnewman-gpsw** [comment]
In the next release the new command will be ```r4V```

**Reply by JerryDiver** [reply]
Thank You!
